platform/src/default.rs

```diff
@@ -109,13 +109,22 @@ impl Platform for DefaultPlatform {
         Ok(issuer_name.len())
     }
 
-    fn get_issuer_sn(&mut self, out: &mut [u8; MAX_SN_SIZE]) -> Result<usize, PlatformError> {
+    fn get_signer_identifier<'a>(
+        &'a mut self,
+        issuer_name: &'a mut [u8; MAX_CHUNK_SIZE],
+        serial_number: &'a mut [u8; MAX_SN_SIZE],
+        _subject_key_identifier: &'a mut [u8; MAX_CHUNK_SIZE],
+    ) -> Result<SignerIdentifier, PlatformError> {
+        let issuer_len = self.get_issuer_name(issuer_name)?;
```

Comment:
Take the cert chain `Root CA -> Intermediate -> Wrapped CSR`. For IssuerAndSerialNumber,

1. Issuer Name should be the the issuer name from `Intermediate`. So the `CA` subject name.
2. Serial number should be `Intermdiate` serial number (which is correct here).

(1) is only correct here because `Root` and `Intermediate` are the same cert in this case (self-signed). For clarity we should probably pull the Issuer Name from the signer (whereas in `get_issuer_name` we get `subject_name` from the issuer.

Reply:
Done